Formula/m/mongo-c-driver@1.rb

```diff
@@ -0,0 +1,45 @@
+class MongoCDriverAT1 < Formula
+  desc "C driver for MongoDB"
+  homepage "https://github.com/mongodb/mongo-c-driver"
+  url "https://github.com/mongodb/mongo-c-driver/archive/refs/tags/1.30.4.tar.gz"
+  sha256 "03e484ff8b382ad0ddf03fbf70e88a82292d753ac2fbf37bac67c2860117b0a9"
+  license "Apache-2.0"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
```

Comment:
this should track 1.x now
```suggestion
    regex(/^v?(1(?:\.\d+)+)$/i)
```

Reply:
Thank you for the suggestion. Updated and squashed into existing commits.